internal/elasticsearch/dashboards.go

```diff
@@ -25,32 +27,55 @@ func (er *ElasticsearchRequest) CreateOrUpdateDashboards() error {
 	if err != nil {
 		return kverrors.Wrap(err, "failed to read dashboard file", "filePath", defaultElasticDashboardFile)
 	}
+
+	var hash string
+	hash, err = utils.CalculateMD5Hash(string(b))
+	if err != nil {
+		log.Error(err, "error calculating hash for elasticsearch dashboard configmap")
+	}
+
 	cm := configmap.New(
 		grafanaCMName,
 		grafanaCMNameSpace,
 		map[string]string{
-			"console.openshift.io/dashboard": "true",
+			"console.openshift.io/dashboard":  "true",
+			constants.TrustedCABundleHashName: hash,
```

Comment:
The constant name is misleading here simply because this is a bad name in the first place. `TrustedCABundleHashName` stands for `logging.openshift.io/hash` and if we use it on both places, we should rename the constant IMHO.

Reply:
I have directly used this `logging.openshift.io/hash` for now. Will probably change the constant name later.